### PR TITLE
Add: cover page, page actions

### PR DESCRIPTION
**Pull request by GoranRibic**

### CODE REVIEW

File: src/react/useCoverPage.ts [hunk 1]
```diff
@@ -20,5 +20,31 @@ export const useCoverPage = (appBridge: AppBridgeTheme): UseCoverPageReturnType
...
+
+                if (event.action === 'delete') {
+                    return null;
+                }
+
+                if (event.action === 'update') {
+                    return { ...previousState, ...event.coverPage };
+                }
+
+                return previousState;
```
Comment (SamuelAlev) [inline src/react/useCoverPage.ts:38]: Switch case?

File: src/react/useDocuments.ts [hunk 2]
```diff
@@ -63,5 +60,5 @@ export const useDocuments = (appBridge: AppBridgeTheme) => {
         };
     }, [appBridge]);
 
-    return [documents];
+    return documents;
```
Comment (SamuelAlev) [inline src/react/useDocuments.ts:63]: Please add `{}`

File: src/types/Emitter.ts [hunk 3]
```diff
@@ -10,6 +10,9 @@ import { Document } from './Document';
 import { DocumentPage } from './DocumentPage';
 import { DocumentGroup } from './DocumentGroup';
 import type { ColorPalette } from './ColorPalette';
+import { CoverPage } from './CoverPage';
```
Comment (SamuelAlev) [inline src/types/Emitter.ts:13]:
```suggestion
import type { CoverPage } from './CoverPage';
```

File: src/react/useCoverPage.ts (hunk 1 above)
Comment (GoranRibic) [reply src/react/useCoverPage.ts]: This is fine in this case, as it CRUD, so it will not expand more than that.

File: src/react/useCoverPage.ts [hunk 4]
```diff
@@ -3,21 +3,52 @@
...
-import type { CoverPage } from '../types';
+import type { CoverPage, EmitterAction } from '../types';
 
 export type UseCoverPageReturnType = {
     coverPage: Nullable<CoverPage>;
 };
 
 export const useCoverPage = (appBridge: AppBridgeTheme): UseCoverPageReturnType => {
     const [coverPage, setCoverPage] = useState<Nullable<CoverPage>>(null);
+    const [fetch, setFetch] = useState(true);
```
Comment (SamuelAlev) [inline src/react/useCoverPage.ts:14]: To avoid collision with `window.fetch` please change it to `isLoading`

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**